src/AzureIoTHub.Portal.Client/Pages/DeviceModels/CreateDeviceModelPage.razor

```diff
@@ -349,13 +501,66 @@
 
             Snackbar.Add("Device model successfully created.", Severity.Success);
 
+
+            // Go back to the list of devices
+            NavigationManager.NavigateTo("device-models");
+        }
+
+        catch (ProblemDetailsException exception)
+        {
+            Error?.ProcessProblemDetails(exception);
+        }
+        finally
+        {
+            isProcessing = false;
+        }
+    }
+    private async Task saveThingType()
```

Comment:
There are code duplications between saveThingType and saveDeviceModel.

Reply:
The duplication is just done in a single line. So I think that not cause a major problem. (To see)